highlight.io/components/QuickstartContent/traces/dot-net-4.tsx

```diff
@@ -254,12 +279,36 @@ public class HighlightTraceProcessor : BaseProcessor<Activity>
                    options.Protocol = ExportProtocol;
                })
                .Build();
+               
+		   _loggerFactory = new LoggerConfiguration()
+                .Enrich.With<HighlightLogEnricher>()
+                .Enrich.FromLogContext()
+                .WriteTo.OpenTelemetry(options =>
+            {
+                options.Endpoint = LogsEndpoint;
+                options.Protocol = OtlpProtocol.HttpProtobuf;
```

Comment:
Nit: should we have this match the other instance format
```
options.Endpoint = new Uri(LogsEndpoint);
options.Protocol = ExportProtocol;
```

Reply:
using a constant for the serilog export, but the `options.Endpoint` here takes a string